internal/domain/irc.go

```diff
@@ -161,7 +161,7 @@ type IRCParser interface {
 
 type IRCParserDefault struct{}
 
-func (p IRCParserDefault) Parse(rls *Release, _ map[string]string) error {
+func (p IRCParserDefault) Parse(rls *Release, vars map[string]string) error {
```

Comment:
This can be reverted to `_` since we do not use the vars here anymore.

Reply:
Done in `5e95dc6`.